moveit_kinematics/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<package>
+  <name>moveit_kinematics</name>
+  <version>0.9.3</version>
+  <description>Package for all inverse kinematics solvers in MoveIt!</description>
+
+  <author email="dave@dav.ee">Dave Coleman</author>
+  <author email="isucan@gmail.com">Ioan Sucan</author>
+  <author email="robot.moveit@gmail.com">Sachin Chitta</author>
+
+  <maintainer email="dave@dav.ee">Dave Coleman</maintainer>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">http://moveit.ros.org</url>
+  <url type="repository">https://github.com/ros-planning/moveit</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>moveit_core</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_depend>actionlib</build_depend>
+  <build_depend>eigen</build_depend>
+  <build_depend>moveit_ros_planning</build_depend>
+
+  <run_depend>moveit_core</run_depend>
+  <run_depend>pluginlib</run_depend>
+  <run_depend>actionlib</run_depend>
+  <run_depend>angles</run_depend>
```

Comment:
I feel that openrave should be listed as a run dependency since the ikfast script won't be able to do anything without  the solution generated by openrave.